### PR TITLE
chore: eslint more views

### PR DESCRIPTION
**Pull request by tyleroooo**
Hitting account info, canvas orderbook, notifications, dialogs

### CODE REVIEW

**Review by yogurtandjam — APPROVED** [review]

File: src/views/notifications/TransferStatusNotification/TransferStatusSteps.tsx [hunk 1]
```diff
@@ -90,26 +93,26 @@ export const TransferStatusSteps = ({ className, status, type }: ElementProps &
       },
     ];
 
-    let currentStep = TransferStatusStep.Bridge;
+    let thisStep = TransferStatusStep.Bridge;
```
Comment (yogurtandjam) [inline src/views/notifications/TransferStatusNotification/TransferStatusSteps.tsx:96]: [suggestion] maybe newCurrentStep or nextStep? to be in line w/ `newSteps`

File: src/views/dialogs/NewMarketMessageDetailsDialog.tsx [hunk 2]
```diff
@@ -139,7 +137,7 @@ export const NewMarketMessageDetailsDialog = ({
                     exchange_config_json
                     {exchangeConfig && <$Tag type={TagType.Number}>{exchangeConfig.length}</$Tag>}
                   </$Text0>
-                  {'['}
+                  [
```
Comment (moo-onthelawn) [inline src/views/dialogs/NewMarketMessageDetailsDialog.tsx:140]: nit: this is sorta hard to read, can we do something like `[${...}]`

File: src/views/notifications/BlockRewardNotification/index.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { useStringGetter } from '@/hooks/useStringGetter';
 
 import { Details } from '@/components/Details';
 import { Icon, IconName } from '@/components/Icon';
+// eslint-disable-next-line import/no-cycle
```
Comment (moo-onthelawn) [inline src/views/notifications/BlockRewardNotification/index.tsx:9]: :'(

**Review by moo-onthelawn — APPROVED** [review]

File: src/views/notifications/BlockRewardNotification/index.tsx (hunk 3 above)
Comment (tyleroooo) [reply src/views/notifications/BlockRewardNotification/index.tsx:9]: I will fix these soon, I struggle to untangle what exactly is happening in here

**Pull request merged**